extensions/git/src/git.ts

```diff
@@ -540,7 +540,8 @@ export class Git {
 		options.env = assign({}, process.env, this.env, options.env || {}, {
 			VSCODE_GIT_COMMAND: args[0],
 			LC_ALL: 'en_US.UTF-8',
-			LANG: 'en_US.UTF-8'
+			LANG: 'en_US.UTF-8',
+			GIT_PAGER: ''
```

Comment:
Are we sure the empty string does the trick?

> To disable pagination for all commands, set core.pager or GIT_PAGER to cat.

From https://git-scm.com/docs/git-config/2.17.0#Documentation/git-config.txt-pagerltcmdgt

Reply:
Probably better to follow recommendation

Reply:
I wonder if it works fine in powershell though...

Reply:
Seems to work fine